rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+- Can un-merge in the future, without submitting a new RFC, if it accumulates enough members to stand on its own.
+
+When a Team or WG is dissolved, the people dissolving it must:
+1. Update its landing page to declare that the it has been dissolved. Reroute visitors accordingly.
+2. Remove the it from the **[TODO] Full list of WGs**  
+3. Remove it from documentation and websites, routing audiences elsewhere
+
+
+## Failsafes
+
+### First Failsafe: Core Working Group
+In case of process failure or broken protocols among the teams, the Core Working Group is responsible for addressing the situation. Their powers and responsibilities are spelled out in the section titled [The Core Working Group](#the-core-working-group).
+
+### "Snakes on the Plane" Failsafe: BDFL
+
+If all other processes and protocols fail, @jbenet will step in and exercise his [beneficially dictatorial](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) prerogatives as the founder and initial creator of these projects.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+- If we don't set the right protocols for creation & dissolution of working groups, we will end up with a lot of confusion about who is responsible for what, meaning balls will get dropped, people will get frustrated, and productive time will be wasted.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+- Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for not choosing them?
+- What is the impact of not doing this?
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+**BLOCKERS**: Questions to resolve before merging this RFC:
+- [x] Research other projects/orgs (5-6). Compare our process against 2-3 of them. Also research how they define the boundaries and responsibilities of Core group.
+  - Done. See document: [Comparison of Open Source Governance Structures](https://docs.google.com/document/d/1vV-8mlqyXsgs_sb9PCuGtqoRlLrvSSjOvJNOZWmZjuI/edit?usp=sharing)
+- [x] Need: Ways to make sure individuals don’t get spread too thin.
+    - Ideas:
+        - People can only captain 1 WG at a time?
+        - Distinguish between product groups and WGs?
+        - Work assignments should happen on Product groups
+    - Addressed: Force 3-person minimum for Teams. Part-time members don't count towards that minimum. This incentivizes teams to get their members to commit, and focus.
+
+
+Questions to resolve through implementation
+- Need to test this process with 3-4 WGs and update the protocol as necessary
+- Investigate how other teams percolate OKRs upward and create an RFCs
+- Need: Support Structure for people Proposing or Captaining a WG
+    - Note: In Rust, the leader of a WG must be part of the core team
+- [ ]  Define: How do OKRs percolate up from sub-teams?
+    - Some options:
+      - The objectives percolate up, becoming KRs in their parent group's OKRs list
```

Comment:
I suspect it will be quite tricky to convert every child's Objective into a parent's KR. In my mind these are fundamentally different things and requirement for such systemic translation may introduce duplication and unnecessary overhead to the entire process. Not saying it can't be done, but the process will be lossy and fuzzy.

Personally I'd go with a rule of thumb stating there should be no "OKR translation" 
(a Team writes OKRs that could be copied&pasted to WG sheet as-is).

Reply:
My understanding of OKRs was that they are defined top-down, see https://rework.withgoogle.com/guides/set-goals-with-okrs/steps/develop-team-OKRs/ also contains a discussion regarding the below bullet point on OKRs at leaves.